### PR TITLE
Create DropDownVariationBehavior.java

### PR DESCRIPTION
**Pull request by helmut8080**
Variant for "drop" for DropdownButton according to bootstrap 4

### CODE REVIEW

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownVariationBehavior.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.Model;
+import org.apache.wicket.util.lang.Args;
+
+import at.lerchen.wicket.bootstrap4.DropDownAlignmentBehavior.Alignment;
+import de.agilecoders.wicket.core.markup.html.bootstrap.behavior.BootstrapBaseBehavior;
+import de.agilecoders.wicket.core.markup.html.bootstrap.behavior.ICssClassNameProvider;
+import de.agilecoders.wicket.core.util.Attributes;
+
+public class DropDownVariationBehavior extends BootstrapBaseBehavior {
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownVariationBehavior.java]: Please add a javadoc what this behavior is good for and when it should be used.

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownVariationBehavior.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+			this.className = className;
+		}
+
+		@Override
+		public String cssClassName() {
+			return className;
+		}
+	}
+
+	private IModel<Variation> alignment;
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownVariationBehavior.java]: Make it `final`

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownVariationBehavior.java (hunk 1 above)
Comment (helmut8080) [reply bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownVariationBehavior.java]: done

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownVariationBehavior.java (hunk 2 above)
Comment (helmut8080) [reply bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownVariationBehavior.java]: done.

**Pull request merged**